### ISSUE TITLE
Rejection 'workspaceFolders'

### ISSUE THREAD

**Issue by kaynetik** [josa42/coc-go#91]
Issue:
```
[coc.nvim] Server languageserver.go failed to start: Launching server "languageserver.go" using command gopls failed.
[coc.nvim]: UnhandledRejection: TypeError: Cannot read property 'workspaceFolders' of undefined
[coc.nvim]: Uncaught exception: Error: process.exit() is not allowed in extension sandbox
[coc.nvim]: /home/anesovic/.config/coc/extensions/node_modules/coc-sql/lib/index.js:153556 client coc abnormal exit with: 7
```

I've tried suggested workarounds. Ie.
`PlugUpdate` & `CocUpdate`
Also, my current `coc-settings.json`:
```
{
    "coc.preferences.formatOnSaveFiletypes": [
        "go",
        "markdown",
        "javascript"
    ],
    "languageserver": {
        "go": {
            "command": "gopls",
            "rootPatterns": ["go.mod", ".vim/", ".git/", ".hg/"],
            "filetypes": ["go"],
            "initializationOptions": {
                "usePlaceholders": true
            },
            "disabled": {
                "workspaceFolders": false
            }
        }
    }
}
```

Could someone recommend anything else I could try?

**Comment by luisdavim** [comment]
this is a duplicate of #90 and #92

**Comment by luisdavim** [comment]
This works for me as a workaround:

```json]  "go.disable": {
    "workspaceFolders": false
  }
```

notice the `disable` instead of `disabled`

**Comment by josa42** [comment]
Fixed in `0.9.2`
